src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
    you now need to use the `IncompleteDsnTestTrait`.
 
  * Make `TransportFactoryTestCase` compatible with PHPUnit 10+
+ * Allow adding DKIM or SMIME signer configuration to the mailer
```

Comment:
this should be improved IMHO, this is fine for frameworkbundle, but this is not accurate for the pert on the mailer component